challenge/src/App.jsx

```diff
@@ -1,45 +1,270 @@
 import React, { Component } from "react";
 import BabyYoda from "./mandalorian/BabyYoda";
-import NumberLogger from './NumberLogger';
+import NumberLogger from "./NumberLogger";
 import strings from "./strings.json";
-import Navbar from './components/Navbar'
-import Fade from 'react-reveal/Fade'
+import Navbar from "./components/Navbar";
+import Fade from "react-reveal/Fade";
 
 class App extends Component {
-    state = {}
-    constructor(props) {
-        super(props);
-    }
-    render() {
-        console.log("Render Method is Working. Redered successfully! I think?")
+  state = {};
+  constructor(props) {
+    super(props);
+    this.state = { currentPage: 0 };
+    this.handlePageChange = this.handlePageChange.bind(this);
+  }
+
+  handlePageChange(index) {
+    this.setState({ currentPage: index });
+  }
+
+  Router(param) {
+    console.log(param.PageRoute);
+    switch (param.PageRoute) {
+      case 0:
+        return (
+          <div>
+            <div
+              style={{ width: "100%", marginTop: "4rem", textAlign: "center" }}
+            >
+              <Fade left cascade>
+                {strings.greeting}
+
+                <form action="https://github.com/HackRU">
+                  <input type="submit" value="Go to GitHub!" />
+                </form>
+
+                <NumberLogger />
+              </Fade>
+            </div>
+            <BabyYoda />
+          </div>
+        );
+      case 1:
         return (
-            <>
-              <Navbar fixed={true} items={['Home', 'Sponsors', 'About', 'Contact us']} />
-              <div style={{ width: "100%", marginTop: "4rem", textAlign: "center" }}>
-                <Fade left cascade>
-                  {strings.greeting}
-
-                  <form action="https://github.com/HackRU">
-                    <input type="submit" value="Go to GitHub!" />
-                  </form>
-
-                  <NumberLogger/>
-                </Fade>
-
-              </div>
-              <BabyYoda />
-            </>
+          <div>
+            <h1 style={{ fontSize: 100, color: "#FFC436" }}>
+              <center>Sponsors Page</center>
+            </h1>
+            <div style={{ backgroundColor: "whitesmoke" }}>
+              <br />
+            </div>
+            <div
```

Comment:
it may also be nice to make a random image component. the random image idea was a nice touch tho

Reply:
I created a random image component in src/components/RandomImage